lib/services/devices/devices-NGSI-v2.js

```diff
@@ -269,7 +269,7 @@ function createInitialEntityNgsi2(deviceData, newDevice, callback) {
  */
 function updateEntityNgsi2(deviceData, updatedDevice, callback) {
```

Comment:
If the same CB API method is used at the end for both create and update cases, why do we need createInitialEntityNgsi2() and updateEntityNgsi2() separate functions? Maybe the can be unified? Or is some additional logic beyond CB request specific in each case?

Reply:
Now both functions are very similar, but it seems they are not following the same logic

Reply:
In this PR on in a later one?

Reply:
IMHO in another PR

Reply:
NTC